app/components/progress_tab/hypertension/control_component.rb

```diff
@@ -4,11 +4,10 @@ class ProgressTab::Hypertension::ControlComponent < ApplicationComponent
   include AssetsHelper
   include ActionView::Helpers::NumberHelper
 
-  attr_reader :control_range, :repository, :current_user
+  attr_reader :repository, :current_user
 
   def initialize(service, current_user)
-    @repository = service.control_rates_repository
-    @control_range = repository.range
```

Comment:
So we don't use the `@control_range` anywhere?

Reply:
`@control_range` is used in the old progress tab, where we show the last 12 months data. Will clean up this later
<img width="258" alt="Screenshot 2022-10-26 at 1 46 38 PM" src="https://user-images.githubusercontent.com/32141642/197972430-108959c9-49bf-4e96-9b66-062d8b33fd3d.png">